integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -357,8 +360,8 @@ public int getattr(String path, FileStat stat) {
       }
       stat.st_mode.set(mode);
       stat.st_nlink.set(1);
-    } catch (InvalidPathException | FileDoesNotExistException e) {
-      LOG.debug("Failed to get info of {}, path is invaid or does not exist", path, e);
+    } catch (FileDoesNotExistException e) {
```

Comment:
Why did we remove the `InvalidPathException` from this catch?

Reply:
Added back, thanks